src/infrastructure/redis-in-memory-sessions/setup-enable-ssl.md

```diff
@@ -10,19 +10,20 @@ platform-version: o11
 
 <div class="info" markdown="1">
 
-Applies to OutSystems self-managed infrastructures. This feature is in Technical Preview.
+Applies to OutSystems self-managed infrastructures.
 
 </div>
 
 <div class="info" markdown="1">
 
-SSL/TLS support is only available in Redis 6.x, and only if the Redis binaries were compiled with this feature enabled.
+SSL/TLS support was only made available starting Redis 6.x, and only if the Redis binaries were compiled with this feature enabled. 
```

Comment:
The original sentence is more accurate grammatically. Technically, there isn't a difference, right?

Reply:
The only difference is that there are new majors 6,7,etc. that now also support SSL/TLS

Reply:
This is basically saying that if the customer wants to use SSL/TLS they can only use versions from 6 and beyond.
Also I don't know if makes sense this test since we only support 7.0.X versions

Reply:
Is the following correct?

SSL/TLS support is only available in Redis 6.x upward, and only if the Redis binaries were compiled with this feature enabled.

Reply:
yes